### PR TITLE
Wait for router to expose pods.

### PR DESCRIPTION
**Pull request by jiripetrlik**
Method waitForScale waits to router to expose pods.

### CODE REVIEW

File: framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/RouterUtil.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
+package org.kie.cloud.openshift.deployment;
```
Comment (sutaakar) [inline framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/RouterUtil.java]: Please add a copyright header.

File: framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/WorkbenchWithKieServerScenarioImpl.java [hunk 2]
```diff
@@ -72,8 +72,6 @@ public WorkbenchWithKieServerScenarioImpl(OpenShiftController openShiftControlle
         envVariables.put(OpenShiftTemplateConstants.IMAGE_STREAM_NAMESPACE, projectName);
         project.processTemplateAndCreateResources(OpenShiftConstants.getKieAppTemplate(), envVariables);
 
-        waitUntilAllPodsAreReady(project);
```
Comment (sutaakar) [inline framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/WorkbenchWithKieServerScenarioImpl.java]: Please also delete private method waitUntilAllPodsAreReady(Project project) as this is the only place where it is used.

**Comment by sutaakar** [comment]
Few minor comments, otherwise looks good.

**Comment by jiripetrlik** [comment]
@sutaakar Thank you for your review. It should be fixed according to review now.

**Comment by sutaakar** [comment]
Looks good.

**Pull request merged**